index.html

```diff
@@ -15,6 +15,11 @@
       </div>
       <nav class="icon">
         <img src="Images/Union.png" alt="Menu-icon" />
+        <ul>
+          <li><a href="#" >Contact</a></li>
+          <li><a href="#" >About</a></li>
+          <li><a href="#" >Portfolio</a></li>
+        </ul>   
       </nav>
     </header>
     <div class="mainContainer">
```

Comment:
The decoration images for the headline are not positioned correctly. Kindly see the design and place them in the right position.

Your design:
![yours-head](https://user-images.githubusercontent.com/35504149/172493042-18d9e1f0-fee7-4653-880f-a03c835e7835.png)

Original design:

![head-o](https://user-images.githubusercontent.com/35504149/172493076-bd48f4f3-449e-486d-9ca6-77c753e1c5aa.png)